torch/csrc/jit/jit_log.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <string>
+
+namespace torch {
+namespace jit {
+
+enum class JitLoggingLevels {
+  OFF,
+  GRAPH_DUMP,
+  INFO,
+  DEBUG,
+};
+
+JitLoggingLevels jit_log_level();
+
+std::ostream& operator<<(std::ostream& out, JitLoggingLevels level);
+
+std::string add_jit_log_prefix(
+    JitLoggingLevels level,
+    const std::string& in_str);
+
+#define JIT_LOG(level, ...)                                                   \
+  if (jit_log_level() != JitLoggingLevels::OFF && jit_log_level() >= level) { \
+    std::cerr << add_jit_log_prefix(level, ::c10::str(__VA_ARGS__));          \
+  }
+
+// print graph after every pass
```

Comment:
Nit: The comment is innacurate - it prints graph whenever it is called, not after every pass (even though the intended use might be to print after each pass).

Reply:
😄 yes, will fix the comment